src/OrchardCore.Commerce.Web/OrchardCore.Commerce.Web.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <CopyRefAssembliesToPublishDirectory>true</CopyRefAssembliesToPublishDirectory>
+    <IsPackable>false</IsPackable>
```

Comment:
Without this, you get warning annotations:
https://github.com/OrchardCMS/OrchardCore.Commerce/actions/runs/4652922954
The warnings are there so if there is a misconfiguration it's not just skipped without a trace.